Dokumentasjon/V1/ClassDiagrams/no.ks.fiks.saksfaser.v1.felles.dokument/classdiagram.pu

```diff
@@ -3,23 +3,17 @@
 
 class Dokument {
     *id: string
-    tittel: string
```

Comment:
Denne bør vurderes da tjenesten er for å vise informasjon til publikum. Dokumenter uten tittel kan være intetsigende.